prettier/prettier.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: prettier
+spec:
+  description: Run the prettier code formatter against provided source code. See https://prettier.io for more information.
+  workspaces:
+  - name: source
+    mountPath: /source
+  params:
+  - name: args
+    type: array
+    description: Prettier arguments used to run Prettier.
+    default:
+      - "--check"
+      - "."
+  steps:
+  - name: check
+    image: elnebuloso/prettier
```

Comment:
v minor thing: We are not very consistent about doing this across our tasks but some tasks allow a user to specify the container image as a param (with a default set). This allows the user to use the same task but with say a different image (say, newer version of prettier)

Reply:
Should I also make the image as a parameter and make this `elnebuloso/prettier` as the default image?

Reply:
yeah, that'd be nice. We can set the default to one of the versions from here: https://hub.docker.com/r/elnebuloso/prettier/tags

Reply:
Sure!